jobrunner/manage_jobs.py

```diff
@@ -125,7 +125,7 @@ def create_and_populate_volume(job):
     # `docker cp` can't create parent directories for us so we make sure all
     # these directories get created when we copy in the code
     extra_dirs = set(Path(filename).parent for filename in input_files.keys())
-    if config.LOCAL_RUN_MODE and job.commit is None:
+    if config.LOCAL_RUN_MODE and not job.commit:
```

Comment:
Do we want to make the equivalent change to the conditional in `create_or_update_jobs.py`?

Reply:
Yup, this is consistent after squashing commits and tidying up.